### DISCUSSION TITLE
Sd3 training guide for winddows?

### DISCUSSION THREAD

**Discussion by GamingDaveUk** [bghira/SimpleTuner#1135] [Q&A]
Following the guide, it didnt know "Source" ok i used call 
then i got to where i need to enter one of 3 commands, nothing for windows.

**Comment by bghira** [comment]
windows is not supported

**Comment by bghira** [comment]
for a good beginner Linux distribution, try Arch Linux or Gentoo.

**Reply by GamingDaveUk** [reply]
I use Ubuntu for my game server (amp) and some other bits, also dietpi on a ...well.... Pi... Lol
But my main pc is used for gaming too, every time I have tried Linux on it I have had issues after issue with games. Dual booting would take too much hd space away. Will use a different program to train sd3.5 medium if there is another that supports it (haven't done training since sdxl and no idea what I used lol)

Thank you for the prompt response.
